js/whitebit.js

```diff
@@ -598,22 +563,13 @@ module.exports = class whitebit extends Exchange {
         //         "deal": "0.00419198" // amount in money
         //     }
         //
-        let timestamp = this.safeTimestamp (trade, 'time');
-        if (timestamp === undefined) {
-            timestamp = this.parse8601 (this.safeString (trade, 'time'));
-        }
+        const timestamp = this.safeTimestamp2 (trade, 'time', 'trade_timestamp');
         const orderId = this.safeString (trade, 'dealOrderId');
         const cost = this.safeString (trade, 'deal');
```

Comment:
I think cost is also the quote_volume here? Should we use safeString2 ? Does the math hold?